uber/models.py

```diff
@@ -194,8 +194,11 @@ def cost_property_names(self):
 
     @property
     def default_cost(self):
```

Comment:
As explained in the docstring, we don't ever want a negative cost, so we make sure it's always at least 0.  This is important for discounts, since the way we've implemented "this age group is free" is by assigning a $999 discount.  (This seems slightly kludgy, but I like the other options less, since it's nice to be able to define in our configspec that ``discount`` is always an integer.)